core/workspace_comment.ts

```diff
@@ -154,7 +154,7 @@ export class WorkspaceComment {
    *   This is not valid if the comment is currently being dragged.
    * @internal
    */
-  getXY(): Coordinate {
+  getRelativeToSurfaceXY(): Coordinate {
```

Comment:
This rename would appear to be a breaking change to the API.  Is that intended?  If so, please update commit message and PR title accordingly.  If not, please (at most) deprecate `getXY` rather than removing it.

Reply:
This method is `@internal`, so I was under the impression we can do whatever we want with it?

Reply:
I'd like to get @rachel-fenichel and/or @maribethb's take on that.  My understanding is that there are some cases where we have previously suggested using (previously) `@private` methods and now treat them as if they had been `@public` for the purpose of deciding what's a breaking change.

If this method is not one we've ever suggested people use then SGTM.

Reply:
I think this is safe--we rarely recommend devs do anything with workspace comments, and I don't remember us ever suggesting people use this. And we use the same name (getRelativeToSurfaceXY) for the same functionality in block/block_svg.